server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -729,6 +729,100 @@
 
   private String adminPassword = "root";
 
+  /** if the startOpenApi is true, we will start OpenApi */
+  private boolean startOpenApi = true;
```

Comment:
the default could be false, be consistent with iotdb-engine.properties

Reply:
fixed